components/employees/date-picker/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from "react";
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+import { Calendar } from "@/components/ui/calendar";
+import { PopoverClose } from "@radix-ui/react-popover";
+import { CalendarDays } from "lucide-react";
+import { Button } from "@/components/ui/button";
+import { format } from "date-fns";
+import { SelectSingleEventHandler } from "react-day-picker";
+
+type DatePickerProps = {
+  onSelect: SelectSingleEventHandler;
+  value?: string;
+  placeholder: string;
+};
+
+const DatePicker: FC<DatePickerProps> = ({ onSelect, value, placeholder }) => {
+  return (
+    <Popover>
+      <PopoverTrigger asChild>
+        <Button
+          variant="popover"
+          className={`w-full text-left font-normal ${!value} && "text-muted-foreground"`}
```

Comment:
text-muted-forground is not from the palete as sanan bhai told discuss it with waseeq bhai if it's intentionally used or mistakenly

Reply:
Whenever we add a new component from shadcn, it uses these default variables. We should not have changed the convention initially, so now we need to change the colors in every new component added.